### PR TITLE
PP-13297 set up controller and template

### PR DESCRIPTION
**Pull request by james-peacock-gds**
- set up controller and template for 'change permission'
- will implement the functionality in a subsequent PR

Screenshot (role headings will be made bolder in a subsequent PR)
Screenshot 2024-11-14 at 16 46 10

### CODE REVIEW

File: app/controllers/simplified-account/settings/team-members/change-permission/change-permission.controller.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+const { findByExternalId } = require('@services/user.service')
+const { response } = require('@utils/response')
+const formatSimplifiedAccountPathsFor = require('../../../../../utils/simplified-account/format/format-simplified-account-paths-for')
```
Comment (DomBelcher) [inline app/controllers/simplified-account/settings/team-members/change-permission/change-permission.controller.js:3]: You can use `require('@utils/...')` here as well

File: app/controllers/simplified-account/settings/team-members/change-permission/change-permission.controller.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+const paths = require('@root/paths')
+
+async function get (req, res, next) {
+  const externalServiceId = req.service.externalId
```
Comment (DomBelcher) [inline app/controllers/simplified-account/settings/team-members/change-permission/change-permission.controller.js:7]: I think just `serviceId` is fine and more in line with usage elsewhere

File: app/controllers/simplified-account/settings/team-members/change-permission/change-permission.controller.js (hunk 1 above)
Comment (james-peacock-gds) [reply app/controllers/simplified-account/settings/team-members/change-permission/change-permission.controller.js]: done

File: app/controllers/simplified-account/settings/team-members/change-permission/change-permission.controller.js (hunk 2 above)
Comment (james-peacock-gds) [reply app/controllers/simplified-account/settings/team-members/change-permission/change-permission.controller.js]: yeah good point

**Review by DomBelcher — APPROVED** [review]
LGTM

**Pull request merged**